doc/v2.rst

```diff
@@ -269,6 +271,11 @@ TensorFlow Serving Predictor
 ``sagemaker.tensorflow.serving.Predictor`` has been renamed to :class:`sagemaker.tensorflow.model.TensorFlowPredictor`.
 (For the previous implementation of that class, see `Deprecate Legacy TensorFlow <#deprecate-legacy-tensorflow>`_).
 
+XGBoost Predictor
+~~~~~~~~~~~~~~~~~
+
+The default seriazlier of ``sagemaker.xgboost.model.XGBoostPredictor`` has been changed from ``NumpySerializer`` to ``LibSVMSerializer``.
+
```

Comment:
I reorganized the docs a bit - I think this would fit better in ["Changes in Default Behavior"](https://github.com/aws/sagemaker-python-sdk/blob/zwei/doc/v2.rst#changes-in-default-behavior)?

Reply:
Also replace `seriazlier` with `serializer`